repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -135,13 +139,26 @@ def process_events(events):
         api.produce(RepositoriesSetupTasks(to_enable=list(to_enable_repos)))
 
 
+def verify_pes_data(path):
+    """Verify if PES data is available if not inhibit upgrade"""
+    if path is None or not os.path.isfile(path):
+        title = 'PES data is invalid (/etc/leapp/files/pes-data.json)'
+        summary = 'Read documentation at: https://access.redhat.com/articles/3664871 for more information ' \
+            'about how to retrieve the files'
+        reporting.report_generic(title=title, summary=summary, severity='high', flags=['inhibitor'])
+        raise StopActorExecutionError(message=title, details={'details': summary})
+
+
 def scan_events(path):
     """ Scan JSON file containing PES events """
     try:
         events = parse_file(path)
-    except (ValueError, KeyError) as error:
-        raise StopActorExecutionError(
-            message='Found invalid or outdated PES data.',
-            details={'details': error, 'hint': 'Consult docs to retrieve updated data.'})
+    except (ValueError, KeyError) as value_error:
+        title = 'PES data is invalid (/etc/leapp/files/pes-data.json)'
```

Comment:
Use "path" variable instead of "/etc/leapp/...", that way if the path changes in future we can fix it in a single place